### PR TITLE
Prepare repo for v2

### PR DESCRIPTION
**Pull request by jpalvarezl**
Closes https://github.com/gnosis/safe-android/issues/542 .

Changes proposed in this pull request:
- Clean-up of source files in preparation for v2
- Added contacts and data modules

@gnosis/mobile-devs

### CODE REVIEW

File: app/src/androidTest/java/io/gnosis/tests/TestApplication.kt [hunk 1]
```diff
@@ -32,7 +32,7 @@ open class TestApplication : MultiDexApplication(), ComponentProvider {
 
     companion object Companion {
         fun mockComponent(): ApplicationComponent =
-            (InstrumentationRegistry.getInstrumentation().targetContext.applicationContext as TestApplication).run {
+            (InstrumentationRegistry.getInstrumentation().targetContext.applicationContext as _root_ide_package_.io.gnosis.tests.TestApplication).run {
```
Comment (rmeissner) [inline app/src/androidTest/java/io/gnosis/tests/TestApplication.kt:35]: `_root_ide_package_` ?

File: app/src/main/java/io/gnosis/safe/HeimdallApplication.kt [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import io.gnosis.safe.di.ComponentProvider
+import io.gnosis.safe.di.components.ApplicationComponent
+import io.gnosis.safe.di.components.DaggerApplicationComponent
+import io.gnosis.safe.di.modules.ApplicationModule
+import org.bouncycastle.jce.provider.BouncyCastleProvider
+import pm.gnosis.crypto.LinuxSecureRandom
+import timber.log.Timber
+import java.security.Security
+
+class HeimdallApplication : MultiDexApplication(), ComponentProvider {
```
Comment (rmeissner) [inline app/src/main/java/io/gnosis/safe/HeimdallApplication.kt:14]: Do we want to keep `HeimdallApplication` (as a tribute to the original app)?

File: app/src/androidTest/java/io/gnosis/tests/TestApplication.kt (hunk 1 above)
Comment (jpalvarezl) [reply app/src/androidTest/java/io/gnosis/tests/TestApplication.kt:35]: just so left over rubbish from refactoring the package name.

**Comment by elgatovital** [comment]
we should eventually get rid of commented out code

**Pull request merged**